### DISCUSSION TITLE
Enforce HTTPS — Unavailable for your site because your domain is not properly configured to support HTTPS

### DISCUSSION THREAD

**Discussion by jaegerev** [community/community#62225] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey guys, I'm trying to set up the website for me and my friend groups little organization.

I have the website up and running on GitHub pages under jaegerev.github.io
We also rent a domain at a German company called strato. There i set a A and a AAAA DNS record to githubs DNS servers for the domain we rented there. 

The DNS check on github pages is successful, as you can see in the following picture though, i cant enforce https for some reason.
![image](https://github.com/community/community/assets/140696321/307c35b0-2f93-43a8-95a7-ffa253a447b6)

I could not find a single file that is accessed via http in the source code of the website. Is there anything special i need to set up on stratos site besides A and AAAA records?

I couldn't find anything in that regard that was of any help on the forums.
Looking forward to your help!

Best Regards,
Robert

**Comment by GhostWr1t3r** [comment]
the provider don't support https configuration you should to get or paid a SSL certificate to enforce https 
OR use a github subdomain 

I'have another solution maybe can help you 
you can use a hosting provider like netlify or vercel and enforce https that maybe can get a ssl certificate

**Comment by jaegerev** [comment, marked as answer]
For anyone finding this question:

I figured it out myself after RCHAMEXE's comment that it may be related to an issue with SSL certificates. What i had to do for it to work with Strato:
1) Remove the free Strato built in SSL certificate from the domain.
2) Add not only a A and AAAA but also a CNAME record on the strato DNS settings as seen in the following picture
![grafik](https://github.com/community/community/assets/140696321/55cb7161-0250-4062-8840-c97ca9821ea1)

**Reply by kaspar-allenbach** [reply]
I added a CNAME record like your example. Doesn't change anything.

**Reply by abedinshanto** [reply]
It's the same for me

**Reply by kaspar-allenbach** [reply]
I figured it out. There were leftover CNAME records from default settings of the DNS entries.
EVERYTHING has to point to github pages.

In my case, there were 5 A records, of which one pointed to my old hoster and some CNAME records which also had old settings.

Those have to go.

**Reply by aprasad36** [reply]
This was my exact issue! Thank you!

**Reply by Kshitij-Banerjee** [reply]
Mine too

**Reply by isaacmachado-dev** [reply]
Verify if you guys have a more than one repo with the same domain link, it seems like an oversight, but it can happen.

**Comment by zeltmanis** [comment]
I used "hostinger" and I had same issue. I deleted all CNAME in DNS that were automatically generated by hostinger. Then, I added just one, my github.io, and it worked.

**Reply by happyhomeind** [reply]
I'm facing the same issue can you please help me with it?

**Reply by isaacmachado-dev** [reply]
Verify if you  have a more than one repo with the same domain link on github-page, it seems like an oversight, but it can happen.

**Reply by securzecom** [reply]
> I'm facing the same issue can you please help me with it?

You may remove all the CNAME, A, AAAA, CAA, and rest all other default records and only keep GitHub records. This will solve the issue. if there are any leftover previous records, then DNS Check might fail from GitHub and SSL is not issued to you.

**Comment by Celltronixpak** [comment]
my issue resolve i chceck the custom name in start type www. then my custom domain save and finally site is online without ssl certificate errors, also now can check hppt inforce option as well, add all your domain critaria by this page settings cname aaaa name a name etc refresh and get started 
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site

**Reply by isk1172** [reply]
This worked for me, Thanks a lot dear friend for posting this solution.
